features/pageobjects/sent.page.js

```diff
@@ -0,0 +1,20 @@
+const Page = require('./page');
+
+const selectors = {
+  sentEmail: 'a[aria-label*="From: test1234512345@yopmail.com"]',

```

Comment:
do not use hardcoded values like this email. your steps and selectors should be reusable if possible
I suggest adding selectors like that 
sentEmail: (emailAddress) => `.//a[contains(@aria-label, "${emailAddress}")]`,

and accordingly

sentEmail(emailAddress) { return $(selectors.sentEmail(emailAddress)); }

and 

When(/^I open the sent email with "(.*)" email $/, async (emailAddress) => {
  await sentPage.sentEmail(emailAddress).click();
});